launcher/Application.cpp

```diff
@@ -690,6 +691,8 @@ Application::Application(int& argc, char** argv) : QApplication(argc, argv)
         m_settings->registerSetting("CloseAfterLaunch", false);
         m_settings->registerSetting("QuitAfterGameStop", false);
 
+        m_settings->registerSetting("Env", QVariant(QMap<QString, QVariant>()));
```

Comment:
Can we rename this setting to `EnvironmentVariables` or `EnvVars`?

Reply:
Env is pretty standard